src/vuestic-theme/vuestic-sass/resources/_mixins.scss

```diff
@@ -11,6 +11,7 @@
   font-size: map-get($display-font-sizes, $element);
   font-weight: bold;
   margin-bottom: 0.5em;
+  line-height: 1;
```

Comment:
Are you sure we need that line here?

We do not plan to fight bootstrap for too long. Its execution is pending.